src/test/java/core/basesyntax/service/impl/FileReaderServiceCsvTest.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.FileReaderService;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FileReaderServiceCsvTest {
+    private static final String FILE_NAME_OK = "src/test/resources/database_ok.csv";
+    private static final String EMPTY_FILE_NAME = "src/test/resources/database_empty.csv";
+    private FileReaderService fileReaderService;
+
+    @Before
+    public void setUp() {
```

Comment:
you are creating `new fileReader` after each test, use `@BeforeClass` instead, do that in all other classes